galley/docker/Dockerfile.galley

```diff
@@ -7,9 +7,14 @@ ARG BASE_VERSION=latest
 # The following section is used as base image if BASE_DISTRIBUTION=default
 FROM docker.io/istio/base:${BASE_VERSION} as default
 
+# Sudoers used to allow debug utilities.
```

Comment:
Can you add this in the base image ( if it is not already there ) ? 

And pretty clear sudoers should only go in the debug images - can we make sure distroless doesn't have it ?

Reply:
yes. Just have another PR to modify the base image - https://github.com/istio/istio/pull/17255